packages/dd-trace/test/appsec/rasp/fs-plugin.spec.js

```diff
@@ -225,6 +225,12 @@ describe('AppsecFsPlugin', () => {
 
       it('should clean up store when finishing op', () => {
         let count = 4
+        // TODO Remove this when node 18 is unsupported or dc-polyfill is fixed&updated
+        //  hack to node 18 and early 20.x
+        //  with dc-polyfill addBind is not enough to force a channel.hasSubscribers === true
+        const onStart = () => {}
+        opStartCh.subscribe(onStart)
```

Comment:
that sounds like a bug in the code that should be fixed in the code, not in the tests, no ?

Reply:
I don't think so, at least for this channel.

This is failing because it is a unit test, but in "real life" each time that this is doing `addBind` we will have a subscription to this channel.

But maybe we need something for `tracing:datadog:express:response:render:start`. I'll take a look.